.idea/workspace.xml

```diff
@@ -1,7 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please revert this file. If we need to update .idea/workspace.xml, it should be done in a separate commit.